src/TinyCiv.Shared/Game/PlayerColor.cs

```diff
@@ -1,9 +1,9 @@
 namespace TinyCiv.Shared.Game;
 
-public enum Color
+public enum PlayerColor
 {
     Red,
     Green,
-    Blue,
-    White
+    Yellow,
+    Purple
```

Comment:
Seems to me that color does not need to be tightly coupled to player, as in, other units could also use it?